pom.xml

```diff
@@ -78,12 +78,6 @@
       <groupId>com.nimbusds</groupId>
       <artifactId>nimbus-jose-jwt</artifactId>
       <version>8.0</version>
-      <exclusions>
```

Comment:
Exclusion can be removed now because the 2.4.5 version appears to override the 2.4.3 version.